babeld/Makefile

```diff
@@ -1,8 +1,6 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
 #
-# Copyright (C) 2007-2009 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
+# Copyright (C) 2007-2021 OpenWrt.org
```

Comment:
Interesting. So the package license is MIT and the Makefile license is GPL. Does that make sense? (Please read this as an open question ...)

Reply:
That is often the case for OpenWrt? @aparcar ?

Reply:
I suspect this is essentially a copy-paste-caused issue. I'm just trying to find out whether it's an issue at all here, now that I got aware of it.

Since it's not really related to the subject of your patch, please ignore this off-topic discussion if it annoys you.

Reply:
No I am also interested what is the correct way. I always thought the "Makefile Code" is GPL since it is OpenWrt and does not rerference the actual programm code... ^^

Reply:
Yea that's my understanding too. You specify the tools actual license via an extra variable